app/views/helpers/Formatters.scala

```diff
@@ -31,12 +31,12 @@ trait DateFormatters {
 
 trait FileFormatters {
 
-  private val kbMin: Long = 1000
-  private val mbMin: Long = 1000000
+  private val kbMin: Long = 1024
```

Comment:
Could you please write tests for these two threshold updates for the fileSize method?